.github/workflows/website.yml

```diff
@@ -45,6 +48,7 @@ jobs:
         uses: r-lib/actions/setup-r@master
         with:
           r-version: 'release'
+          http-user-agent: "R/4.0.0 (ubuntu-16.04) R (4.0.0 x86_64-pc-linux-gnu x86_64 linux-gnu) on GitHub Actions"
```

Comment:
I've noticed you use `ubuntu-16.04` here. The `ubuntu-latest` runner that we use in this action is actually Ubuntu 18.04. Does that affect R packages?

Reply:
Good catch! that was me foolishly copy/pasting the UA string from another example.

Reply:
Actually, looking at the latest version of the standard check, it seems they don't need a modified UA string any more. https://github.com/r-lib/actions/blob/master/examples/check-standard.yaml